enterprise/frontend/src/embedding-sdk/index.ts

```diff
@@ -11,6 +13,10 @@ import "sdk-ee-plugins";
 // Imports which are only applicable to the embedding sdk, and not the new iframe embedding.
 import "sdk-specific-imports";
 
+// Enable SDK mode as we are in the SDK bundle
+// This applies to SDK derivatives such as new iframe embedding.
+EMBEDDING_SDK_CONFIG.isEmbeddingSdk = true;
```

Comment:
We import the "embedding-sdk" entry point from `SdkIframeEmbedRoute.tsx` in new iframe embedding too, so this field gets set to `true` for both for the SDK and new iframe embedding.

Reply:
@npretto Would setting it by function e.g. `setIsEmbeddingSdk(true)` be more explicit in your opinion?

Reply:
Fixed in https://github.com/metabase/metabase/pull/60349/commits/5082535584f36ee3b0685053f7a39c9894c0d9a0, thanks!